server/src/main/resources/application.properties

```diff
@@ -123,3 +123,9 @@ conductor.workflow-execution-lock.type=noop_lock
 # Additional modules for metrics collection exposed to Datadog (optional)
 management.metrics.export.datadog.enabled=${conductor.metrics-datadog.enabled:false}
 management.metrics.export.datadog.api-key=${conductor.metrics-datadog.api-key:}
+
+
+conductor.event-queues.kafka.truststorePath=/app/truststore/client.truststore.jks
+conductor.event-queues.kafka.truststorePassword=default
+conductor.tasks.kafka-publish.truststorePath=/app/truststore/client.truststore.jks
+conductor.tasks.kafka-publish.truststorePassword=default
```

Comment:
Since we are not mounting a trust store directly, these will be the default location and credentials.